tests/unit/OrgPerson.spec.ts

```diff
@@ -327,6 +329,7 @@ describe('Org/Person component', () => {
 
   it('Emits "reset" event when clicking Cancel button', async () => {
     const wrapper = createComponent(validOrgData, 0, null)
+    wrapper.vm.applyValidation()
```

Comment:
You don't need to wait a tick after this (as you do in the main code)?

I'd be tempted to move the "nextTick" in the main code to the bottom of applyValidation() and then await it (just so it's self-contained).

Reply:
Ok, good idea. Will try that

Reply:
It wasn't required in the test, probably because it awaits a process after updating the input fields.

Reply:
Added them for consistency though, as not all tests had similar processes.

Reply:
I was thinking you'd move the "nextTick" inside "applyValidation".

The principle behind this is that the caller shouldn't be concerned about calling nextTick after calling applyValidation: it's one call (though it will have to awaited). If applyValidation requires a short delay to properly complete then it should be done inside applyValidation.

This is [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns), where each chunk of code should need to know as little as possible about other code.

Reply:
I did move it inside. 

The test suites however still require this (some of them), i just added to all for consistency.

Reply:
Oh I'm sorry, I didn't notice you updated this already.

Probably the reason some tests require this is because they need to await applyValidation.

Reply:
Side note: I've found (to my surprise) that most things that don't look right can be explained. So, now, if something doesn't look right, I look for the explanation... Typically it's because I'm doing something incorrectly 😆 

Moral of the story: if it doesn't look right, try to figure out why.

Reply:
Found that i didn't need the NextTick inside, once we awaited the call